data/data/vsphere/bootstrap/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "vsphere_virtual_machine" "vm" {
+  name             = "bootstrap"
+  resource_pool_id = var.resource_pool
+  datastore_id     = var.datastore
+  num_cpus         = 4
+  memory           = 16384
+  guest_id         = var.guest_id
+  folder           = var.folder
+  enable_disk_uuid = "true"
+
+  wait_for_guest_net_timeout  = "0"
+  wait_for_guest_net_routable = "false"
```

Comment:
should change all these to actual types instead of string.

Reply:
ack